### PR TITLE
Initial commit for React UI Components Project 

### PR DESCRIPTION
**Pull request by jthicks91**

### CODE REVIEW

File: Project-I-Social-Card/social-card/src/App.css [hunk 1]
```diff
@@ -2,3 +2,14 @@
...
+
+.App {
+  width: 100%;
+  margin: 0 auto;
+  margin-top: 200px;
+  border: 1px solid black;
+  border-radius: 5%;
+  max-width: 500px;
+  padding: 10px 20px;
+  box-shadow: 2px 2px 4px 5px rgb(95, 95, 95);
```
Comment (kyranmccann) [inline Project-I-Social-Card/social-card/src/App.css:14]: look at that fancy box shadow. niiiice

File: Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js [hunk 2]
```diff
@@ -1,2 +1,19 @@
-import React from 'react';
-import './Card.css';
+import React from "react";
+
+import "./Card.css";
+
```
Comment (kyranmccann) [inline Project-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js:4]: the card.css only needs to be imported into one component (the top-level of the card)

File: Project-I-Social-Card/social-card/src/App.js [hunk 3]
```diff
@@ -1,23 +1,18 @@
...
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
+    <React.Fragment>
+      <div className="App">
+        <HeaderContainer />
+        <CardContainer />
```
Comment (kyranmccann) [inline Project-I-Social-Card/social-card/src/App.js:13]: footer?

**Review by kyranmccann — APPROVED** [review]

File: Project-I-Social-Card/social-card/src/App.js (hunk 3 above)
Comment (jthicks91) [reply Project-I-Social-Card/social-card/src/App.js:13]: i thought it was a stretch....

Comment (kyranmccann) [reply Project-I-Social-Card/social-card/src/App.js:13]: It was. The rest of it is just so nice I think you could have knocked it out!

Comment (jthicks91) [reply Project-I-Social-Card/social-card/src/App.js:13]: Thank you and challenge accepted!

**Pull request merged**